docs/en/latest/config.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": 0.1,
+    "sidebar": [
+      {
+        "type": "category",
```

Comment:
The type is incorrect, see: https://github.com/apache/apisix/blob/2707fda3eb7b23d995ac23ecf4a17000dd765855/docs/en/latest/config.json#L164 as an example

Reply:
oh yes , I forgot, it done now